### PR TITLE
[Serializer] Fix Groups tests.

### PR DESCRIPTION
**Pull request by dunglas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

### CODE REVIEW

File: src/Symfony/Component/Serializer/Tests/Annotation/GroupsTest.php [hunk 1]
```diff
@@ -19,7 +20,7 @@
 class GroupsTest extends \PHPUnit_Framework_TestCase
 {
     /**
-     * @expectedException \InvalidArgumentException
+     * @expectedException InvalidArgumentException
```
Comment (fabpot) [inline src/Symfony/Component/Serializer/Tests/Annotation/GroupsTest.php]: Looks wrong to me. IIRC, PHPUnit does not resolve class names, we must use FQCN

Comment (dunglas) [inline src/Symfony/Component/Serializer/Tests/Annotation/GroupsTest.php]: It seems to work with the last version but I'll change it anyway.

Comment (dunglas) [inline src/Symfony/Component/Serializer/Tests/Annotation/GroupsTest.php]: Done.

**Comment by fabpot** [comment]
Thank you @dunglas.

**Pull request merged**